test/util/e2e.go

```diff
@@ -232,3 +230,23 @@ func RestartKueueController(ctx context.Context, k8sClient client.Client) {
 	kcmKey := types.NamespacedName{Namespace: "kueue-system", Name: "kueue-controller-manager"}
 	rolloutOperatorDeployment(ctx, k8sClient, kcmKey)
 }
+
+func WaitForActivePodsAndTerminate(ctx context.Context, k8sClient client.Client, restClient *rest.RESTClient, cfg *rest.Config, namespace string, activePodsCount, exitCode int) {
+	activePods := make([]corev1.Pod, 0)
+	pods := corev1.PodList{}
+	gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
```suggestion
	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
```

Reply:
@mszadkow Additionally, could you open a PR to add offset to other functions in this `test/util/e2e.go`?